src/tiledimage.js

```diff
@@ -1515,7 +1515,7 @@ function loadTile( tiledImage, tile, time ) {
     tiledImage._imageLoader.addJob({
         src: tile.url,
         loadWithAjax: tile.loadWithAjax,
-        ajaxHeaders: tile.ajaxHeaders,
+        ajaxHeaders: tiledImage.ajaxHeaders,
```

Comment:
Now that I'm taking a closer look at the ajaxHeaders functionality, it looks like the tile source has a `getTileAjaxHeaders` which allows for different headers at the tile level. I'm not sure who, if anyone, is using that functionality, but I don't think we should break it with this pull request.

Currently, getTileAjaxHeaders is called by the tiledImage when the tile is created, and the tiledImage's headers are combined with the tile's at that time. I propose that instead we load just the tile's headers into the tile at creation, and combine them with the tiledImage's headers here. That way we can support both scenarios (per-tile headers, and tiledImage headers that change dynamically).